src/library/handlers/metricHandler.ts

```diff
@@ -72,8 +69,8 @@ export class MetricHandler {
         this._nativePerformance.enable(this.isNativePerformance, this.disabledExtendedMetrics);
     }
 
-    public flush(options?: FlushOptions) {
-        this._batchProcessor.triggerSend(options ? options.isAppCrashing : false);
+    public flush(isAppCrashing?: boolean) {
```

Comment:
Personally, I like the previous implementation (passing a FlushOptions) options object as it allows for any future expansion without the need to add additional arguments -- even though you are only using the one today.

eg. in the future we might want to add things like timeout, retry, synchronous etc